server.js

```diff
@@ -23,6 +34,43 @@ server.addHook("onRequest", async (req, reply) => {
   };
 });
 
+server.addHook("onSend", function (request, reply, payload, next) {
+  if (process.env.ENABLE_REQUEST_LOGGING === "true") {
```

Comment:
@arafatalim Please add this variable to `.env.example`

Reply:
ok @rajeshn95